letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -488,13 +488,56 @@ BootstrapRpmCommon() {
   BootstrapRpmCommonBase "$python_pkgs"
 }
 
+# If new packages are installed by BootstrapRpmPython3 below, this version
+# number must be increased.
+BOOTSTRAP_RPM_PYTHON3_LEGACY_VERSION=1
+
+BootstrapRpmPython3Legacy() {
+  # Tested with:
+  #   - CentOS 6
+
+  InitializeRPMCommonBase
+
+  if ! $TOOL list rh-python36 >/dev/null 2>&1; then
+    echo "To use Certbot, packages from the SCL repository need to be installed."
+    if ! $TOOL list centos-release-scl >/dev/null 2>&1; then
+      error "Enable the SCL repository and try running Certbot again."
+      exit 1
+    fi
+    if [ "$ASSUME_YES" = 1 ]; then
+      /bin/echo -n "Enabling the SCL repository in 3 seconds..."
```

Comment:
Would it be helpful to say `Enabling the SCL repository in 3 seconds... (Press Ctrl-C to cancel)`? (I assume almost all Certbot users know that, but maybe a handful don't?)

Reply:
Added. Also updated `rpm_common_base.sh` on the similar message for consistency.